frontend/public/components/monitoring/metrics.tsx

```diff
@@ -438,12 +453,21 @@ export const QueryBrowserPage = withFallback(() => {
       </div>
       <div className="row">
         <div className="col-xs-12">
-          <QueryBrowser
-            defaultTimespan={30 * 60 * 1000}
-            disabledSeries={_.map(queries, 'disabledSeries')}
-            onDataUpdate={onDataUpdate}
-            queries={_.map(queries, 'query')}
-          />
+          {queryStrings.join('') === ''
+            ? <div className="query-browser__wrapper graph-empty-state">
+              <EmptyState variant={EmptyStateVariant.full}>
+                <EmptyStateIcon size="sm" icon={ChartLineIcon} />
+                <Title size="sm">No Query Entered</Title>
+                <EmptyStateBody>Enter a query in the box below to explore metrics for this cluster</EmptyStateBody>
+                <Button onClick={insertExampleQuery} variant="primary">Insert Example Query</Button>
```

Comment:
The different button styles are a little odd, but @sg00dwin is working on converting all buttons to PF4.